<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #152028;

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>70</width>
     <height>780</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1F2933;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="VaultButton">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>210</y>
      <width>91</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#VaultButton {
	background: #1F2933;
	border: none;
}

QPushButton#VaultButton:hover {
	background:  #152028;;
	border: none;
}

QPushButton#VaultButton:pressed {
	border: none;
	background: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../CredentialMenu/files.qrc">
      <normaloff>:/Images/SideBar/vault.png</normaloff>:/Images/SideBar/vault.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>71</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EFEFEF;
QPushButton#SettingsButton {
	background: #1F2933;
	border: none;
}
QPushButton#SettingsButton:hover {
	background:  #152028;;
	border: none;
}

QPushButton#SettingsButton:pressed {
	border: none;
	background: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../CredentialMenu/files.qrc">
      <normaloff>:/Images/SideBar/settings.png</normaloff>:/Images/SideBar/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>58</width>
      <height>58</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ImportExportOptionsButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>81</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#ImportExportOptionsButton {
	background: #1F2933;
	border: none;
}
QPushButton#ImportExportOptionsButton:hover {
	background:  #152028;;
	border: none;
}

QPushButton#ImportExportOptionsButton:pressed {
	border: none;
	background: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../CredentialMenu/files.qrc">
      <normaloff>:/Images/SideBar/export.png</normaloff>:/Images/SideBar/export.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>0</y>
     <width>991</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #152028;
color: white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="ImportExportFrame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>161</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid Black;
background: #152028;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="ExportAllButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1F2933;
border: 2px solid #323F4B;
color: white;</string>
    </property>
    <property name="text">
     <string>Export All</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="SettingsFrame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>711</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1F2933;
border: 2px solid white;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="GeneratorButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #152028;
border: 2px solid #323F4B;
color: white;</string>
    </property>
    <property name="text">
     <string>Password Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GeneratorButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #152028;
border: 2px solid #323F4B;
color: white;</string>
    </property>
    <property name="text">
     <string>Reset Password</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1F2933;
border: 2px solid #1F2933;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>31</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #152028;
color: white;</string>
     </property>
     <item>
      <property name="text">
       <string>Minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hours</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>33</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #152028;
color: white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Automatically Log Out After</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>38</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>radioButton</zorder>
    <zorder>comboBox</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit</zorder>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>GeneratorButton</zorder>
   <zorder>GeneratorButton_2</zorder>
  </widget>
  <zorder>SettingsFrame</zorder>
  <zorder>label_8</zorder>
  <zorder>frame</zorder>
  <zorder>ImportExportFrame</zorder>
 </widget>
 <resources>
  <include location="../CredentialMenu/files.qrc"/>
 </resources>
 <connections/>
</ui>
